app/src/main/java/app/insti/adapter/ComplaintsAdapter.java

```diff
@@ -297,9 +297,9 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder viewHolder, int po
         List<User> userList = complaintList.get(position).getUsersUpVoted();
         for (User user : userList) {
             if (user.getUserID().equals(userID))
-               complaintList.get(position).setVoteCount(1);
+               complaintList.get(position).setComplaintUpvoted(true);
             else
```

Comment:
IIRC you no longer should need to do this since the server would send the value of `is_subscribed`

Reply:
My bad no it shouldn't. I only removed it from the ComplaintFragment and forgot about this. I have fixed it though and added a feature to mark the complaints the users have upvoted in the ComplaintsAdapter.